docs/guides/debug-qiskit-runtime-jobs.ipynb

```diff
@@ -0,0 +1,616 @@
+      "text/plain": [
+       "<Figure size 1123.61x535.111 with 1 Axes>"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "def generate_circuit(n_qubits, n_layers):\n",
+    "    r\"\"\"\n",
+    "    A function to generate a pseudo-random a circuit with ``n_qubits`` qubits and\n",
+    "    ``2*n_layers`` entangling layers of the type used in this notebook.\n",
+    "    \"\"\"\n",
+    "    # An array of random angles\n",
+    "    angles = [\n",
+    "        [random.random() for q in range(n_qubits)] for s in range(n_layers)\n",
+    "    ]\n",
+    "\n",
+    "    qc = QuantumCircuit(n_qubits)\n",
+    "    qubits = list(range(n_qubits))\n",
+    "\n",
+    "    # do random circuit\n",
+    "    for layer in range(n_layers):\n",
+    "        # rotations\n",
+    "        for q_idx, qubit in enumerate(qubits):\n",
+    "            qc.rz(angles[layer][q_idx], qubit)\n",
+    "\n",
+    "        # cx gates\n",
+    "        control_qubits = (\n",
+    "            qubits[::2] if layer % 2 == 0 else qubits[1 : n_qubits - 1 : 2]\n",
+    "        )\n",
+    "        for qubit in control_qubits:\n",
+    "            qc.cx(qubit, qubit + 1)\n",
+    "\n",
+    "    # undo random circuit\n",
+    "    for layer in range(n_layers)[::-1]:\n",
+    "        # cx gates\n",
+    "        control_qubits = (\n",
+    "            qubits[::2] if layer % 2 == 0 else qubits[1 : n_qubits - 1 : 2]\n",
+    "        )\n",
+    "        for qubit in control_qubits:\n",
+    "            qc.cx(qubit, qubit + 1)\n",
+    "\n",
+    "        # rotations\n",
+    "        for q_idx, qubit in enumerate(qubits):\n",
+    "            qc.rz(-angles[layer][q_idx], qubit)\n",
+    "\n",
+    "    return qc\n",
+    "\n",
+    "\n",
+    "# Generate a random circuit\n",
+    "qc = generate_circuit(6, 3)\n",
+    "isa_qc = pm.run(qc)\n",
+    "\n",
+    "qc.draw(\"mpl\", idle_wires=0)"
```

Comment:
I might be overthinking it but maybe someone that's not very familiar with transpiler passes and ISA could wonder why `pm.run` is needed, especially since then we are drawing the `qc` circuit. Maybe we can add an inline comment explaining that the pass is used to convert the abstract circuit to an equivalent ISA circuit.